tests/cpp/test_evaluator.cpp

```diff
@@ -478,6 +479,18 @@ TEST_F(ExprEvalTest, TensorMetaData) {
   checkIntValue(evaluator, size1, 128L);
   checkIntValue(evaluator, stride0, 128L);
   checkIntValue(evaluator, stride1, 1L);
+
+  std::ostringstream ss;
+  DebugStreamGuard dsg(ss);
+
+  evaluator.print();
+  {
+    // Now bind a PrecomputedValues and print
+    PrecomputedValues pv(&fusion);
+    evaluator.bindPrecomputedValues(&pv);
+    pv.bindInputs(KernelArgumentHolder::createKernelArgumentHolder({a}));
+    evaluator.print();
+  }
```

Comment:
Is this just to verify things don't crash when printing? FWIW, we can https://github.com/NVIDIA/Fuser/blob/522779f20cf2cc06fde579dab1758a9560c1721b/tests/cpp/test_gpu3.cpp#L6827 to capture stdout if you intended to check the printed strings as well.

Reply:
Yes, I just wanted to check that we do not hit an error when printing. But now that you mention it I will use this part to check that we are printing the variables we expect to be bound.